packages/plugin-ext/src/main/browser/plugin-frontend-contribution.ts

```diff
@@ -15,21 +15,56 @@
 // *****************************************************************************
 
 import { injectable, inject } from '@theia/core/shared/inversify';
-import { CommandRegistry, CommandContribution } from '@theia/core/lib/common';
+import { CommandRegistry, CommandContribution, Command } from '@theia/core/lib/common';
 import { OpenUriCommandHandler } from './commands';
 import URI from '@theia/core/lib/common/uri';
+import { TreeViewWidget } from './view/tree-view-widget';
+import { CompositeTreeNode, Widget, codicon } from '@theia/core/lib/browser';
+import { TabBarToolbarContribution, TabBarToolbarRegistry } from '@theia/core/lib/browser/shell/tab-bar-toolbar';
+import { PluginViewWidget } from './view/plugin-view-widget';
 
 @injectable()
-export class PluginApiFrontendContribution implements CommandContribution {
+export class PluginApiFrontendContribution implements CommandContribution, TabBarToolbarContribution {
 
     @inject(OpenUriCommandHandler)
     protected readonly openUriCommandHandler: OpenUriCommandHandler;
 
+    static readonly COLLAPSE_ALL_COMMAND = Command.toDefaultLocalizedCommand({
+        id: 'treeviews.collapseAll',
+        iconClass: codicon('collapse-all')
```

Comment:
```suggestion
        iconClass: codicon('collapse-all'),
        label: nls.localizeByDefault('Collapse All')
```

Otherwise we don't get any hover info on the toolbar button.